CHANGELOG.md

```diff
@@ -333,6 +339,7 @@ Improvements:
 
 ## [0.1.0] - 2019-05-14
 
+[unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.8.0...HEAD
```

Comment:
Is this the correct repository in the URL?

Reply:
Of course... not. :upside_down_face:

It would be nice if there would be a tool which checks the correctness of the changelog. Unfortunately, I could not find something suitable.